griptape/common/prompt_stack/messages/delta_prompt_stack_message.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+from typing import Optional
+
+from attrs import define, field
+
+from griptape.common.prompt_stack.contents.text_delta_prompt_stack_content import TextDeltaPromptStackContent
+
+
+from .base_prompt_stack_message import BasePromptStackMessage
+
+
+@define
+class DeltaPromptStackMessage(BasePromptStackMessage):
```

Comment:
Yeah I went back and forth on this. I didn't love putting the `Delta`/`Chunk`/etc at the end because it breaks the naming convention of its parent class. `PartialPromptStackMessage` works but is inconsistent with the usage of `Delta` in the Prompt Stack Content classes. So I could update those to `Partial`, but `Delta` feels better there.

What do you think?